rr_ml/nodes/image_label_tools/extract_images.py

```diff
@@ -0,0 +1,51 @@
+import os
+import random
+
+import rospy
+import rosbag
+from cv_bridge import CvBridge
+import cv2
+
+from . import util
+
```

Comment:
It's an important caveat that this only extracts a random subset of the images in the bag (unless you set num_images to be the number of images in the bag), I would add that in as a comment somewhere